sphinx/source/docs/user_guide/plotting.rst

```diff
@@ -113,12 +113,10 @@ Composition
 ~~~~~~~~~~~
 
 Bokeh plots are composed of "glyphs", which are semi-primitive visual markers.
-Each glyph has specified parameters for placement and styling.
-You can refer to the
-`Bokeh Glyph Quick Reference <http://bokeh.pydata.org/docs/glyphs_ref.html>`_
-to see all the glyphs that are currently supported, and to the
-`Glyph Functions <http://bokeh.pydata.org/docs/reference.html#glyphs-functions>`_
-section to see how they are configured for the ``plotting.py`` interface.
+Each glyph has specified parameters for placement and styling. You can refer
+to :ref:`bokeh.models.glyphs` and :ref:`bokeh.models.markers` to see all the
+glyphs that are currently supported, and to :ref:`bokeh_plotting_plots`
```

Comment:
Is there a reason for having `bokeh.x.y` and `bokeh_x_y`, or is that just history?

Otherwise LGTM.